cli/azd/pkg/devcenter/prompter.go

```diff
@@ -66,6 +66,10 @@ func (p *Prompter) PromptProject(ctx context.Context, devCenterName string) (*de
 		return nil, err
 	}
 
+	if len(writeableProjects) == 0 {
+		return nil, fmt.Errorf("no dev center projects have been found")
```

Comment:
Is it more concise to omit "have been", i.e.:
```suggestion
		return nil, fmt.Errorf("no dev center projects found")
```